ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,249 @@
+﻿[[!meta title="Getting Involved"]]
+
+## How can I get involved in the OCF?
+
+Glad you asked! Students from all backgrounds are welcome to join staff. We
+don’t interview or have a selection process for joining—you simply have to
+express interest and show up to our weekly meetings. We don't impose a specific
+time commitment; some volunteers work many hours per week, others contribute
+occasionally. If you want to learn more and be an active participant, there are
+multiple ways to get started:
+
+### Stay up-to-date
+
+We use a mailing list ([ocf.io/announce][announce]) to announce weekly meetings
+and recap past meetings. If you’re joining in the middle of the semester or
+missed a meeting, you can also read the mailing list archives to see what we
+went over in the past. This is a low volume mailing list (2 emails per week)
+and you can unsubscribe at any time.
+
+[[Read more about other mailing lists available to join|doc
+staff/mailing-lists]]
+
+### In-person meetings
+
+We hold two meetings every Monday:
+
+#### Board of Directors (BoD)
```

Comment:
Put staff meeting before BoD, since it's more geared toward newcomers? (Or is this a deliberate move to encourage BoD participation?)

Reply:
I can see both sides, but I think it's best to put BoD first because it is held in time before staff meeting. It's less confusing this way.

Reply:
On the other hand, though, in order of importance to newcomers, staff meeting would come before BoD...